exercises/hello-world/examples/success-standard/src/HelloWorld.hs

```diff
@@ -0,0 +1,4 @@
+module HelloWorld (hello) where
+
+hello :: String
+hello = "Hello, World!"
```

Comment:
I worry a little that maybe this way the student will be able to pass the tests without finding the source file that needs to be changed.